templates/docker-compose/restore.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+. .env
+LAST_BACKUP_DIR="backup/$(ls -1t backup/ | head -n1)"
+LAST_BACKUP_FILE=$(ls -1 ${LAST_BACKUP_DIR} | grep 'app-.*gz')
+
+
+TIMESTAMP=$(date +%Y-%m-%d-%H-%M-%S-%s)
+TEMP_BACKUP_DIR="/tmp/tmp-backups/${TIMESTAMP}"
+docker exec "${WEB_SERVICE_NAME}" bash -c "mkdir -p ${TEMP_BACKUP_DIR}"
+docker cp "${LAST_BACKUP_DIR}/${LAST_BACKUP_FILE}" "${WEB_SERVICE_NAME}:${TEMP_BACKUP_DIR}/"
+docker exec "${WEB_SERVICE_NAME}" bash -c "cd /supportpal/app-manager && php artisan app:restore ${TEMP_BACKUP_DIR}/${LAST_BACKUP_FILE} --no-verify --force"
```

Comment:
This cannot be merged into `master` until v4.2.0 is released. Is there any reason this is different to `docker-monolithic/backup.sh` which does work on v4.1.2?

Reply:
Seems like monolithic won't actually work on `4.2.0` as `app:restore` is obsolete and errors. I think it would be best if this did something like, in psuedo code, if config/release.php is >= v4.2.0 then use `app-manager` else use the base directory. Then we can merge this now, and it'll still work after v4.2.0 is released.